src/Http/Livewire/LivewireDatatable.php

```diff
@@ -609,10 +640,10 @@ public function getSortString()
                 return Str::before($column['select'], ' AS ');
                 break;
 
-             default:
+            default:
                 return $dbTable == 'pgsql' || $dbTable == 'sqlsrv'
-                    ? new Expression('"' . $column['name'] . '"')
```

Comment:
@kountouris7 Why are we changing this? It only applies fo `postgres` and `sql server`. Have we tested with these DBs?

Reply:
@michalisantoniou6 all the tests in livewire-datatables that interact with the DB, are using pgsql by default.

Reply:
@kountouris7 Are you sure they use postgres? It doesn't sound accurate. I don't have postgres installed locally and I'm able to run the tests. 

Also, why are we changing this? Why are we deleting `new Expression()`? Do we need to make this change for multisort to work? I'm positive there was a reason the original authors wrote `new Expression()` here. I'm not sure we can just remove it.

Reply:
You are right. Got confused and thought something else.